weave

```diff
@@ -95,6 +95,8 @@ where <peer>     = <ip_address_or_fqdn>[:<port>]
       <endpoint> = [tcp://][<ip_address>]:<port> | [unix://]/path/to/socket
       <peer_id>  = <nickname> | <weave internal peer ID>
       <mode>     = consensus[=<count>] | seed=<mac>,... | observer
+
+To troubleshoot and visualize Weave networks, try Weave Cloud: http://www.weave.works/product/cloud/
```

Comment:
I think as "a word from our sponsor" it's fine to blab about vaguely-related things.
So would we explictly say something like "Weave Net is brought to you by the makers of ..." ?